src/components/AudioTrack/layouts/Row.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="row-track flex flex-row" :class="`size-${size}`">
+    <AudioThumb
+      class="flex-shrink-0"
+      :class="isSmall ? 'w-20 mr-4' : 'w-30 mr-6'"
+      :audio="audio"
+    />
+    <div class="flex" :class="isSmall ? 'flex-row gap-8' : 'flex-col gap-4'">
+      <div class="flex-shrink-0">
+        <p class="font-heading font-semibold text-2xl">{{ audio.title }}</p>
+
+        <div
+          class="flex leading-snug text-dark-charcoal-70 mt-2"
+          :class="
+            isSmall ? 'flex-col gap-2' : 'flex-row items-center justify-between'
+          "
+        >
+          <div class="part-a">
+            <i18n
+              tag="span"
+              class="font-semibold leading-snug"
+              path="audio-track.creator"
+            >
+              <template #creator>{{ audio.creator }}</template>
+            </i18n>
+            <span v-if="!isSmall">
+              • {{ audio.duration }} • {{ audio.category }}
```

Comment:
Duration needs to be converted from milliseconds to min:sec.
And category needs to be translated. Or is it passed here already translated?

Reply:
Converted and formatted duration in f11dfae. Category translated in f5e63b2.